### DISCUSSION TITLE
Add a way to let moderators change the title and or category

### DISCUSSION THREAD

**Discussion by shoXyRafael** [Chatterino/chatterino2#3432] [Ideas] [answered]
As the title says, the idea is that there is a tab with stream info in the settings where you can view the current stream title and category and these can also be changed. In my opinion it is much more faster because you do not always have to go to the moderation view to change it there.

**Comment by zneix** [comment, marked as answer]
This is not possible - to be able to change title / game for channel other than your own you'd need to have streamer's access token ([reference](https://dev.twitch.tv/docs/api/reference#modify-channel-information), although it isn't mentioned on docs site, the `broadcaster_id` needs to match user ID in the provided access token), which isn't realistically possible because in Chatterino you only log in with your own account and it only knows your user access token. I'd recommend using a chatbot with !setgame / !settitle commands instead. I'm sorry, but this is just not possible to do.
